roles/openshift_hosted/defaults/main.yml

```diff
@@ -30,3 +30,7 @@ openshift_hosted_routers:
 openshift_hosted_router_certificate: {}
 openshift_hosted_registry_cert_expire_days: 730
 openshift_hosted_router_create_certificate: False
+
+os_firewall_allow:
+- service: Docker Registry Port
+  port: 5000/tcp
```

Comment:
@sdodson @mtnbikenc @ewolinetz do we have a list of allocated ports for OCP? So we don't reuse a port already used by a different service.